tests/manager/manager-ctor-auto_encryption-001.phpt

```diff
@@ -0,0 +1,83 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): auto encryption options
+--FILE--
+<?php
+
+$baseOptions = [
+    'keyVaultNamespace' => 'admin.dataKeys',
+    'kmsProviders' => ['aws' => (object) ['accessKeyId' => 'abc', 'secretAccessKey' => 'def']]
+];
+
+$tests = [
+    [],
+    ['keyVaultClient' => new MongoDB\Driver\Manager()],
+    ['schemaMap' => [
+        'default.default' => [
+            'properties' => [
+                'encrypted_objectId' => [
+                    'encrypt' => [
+                        'keyId' => [
+                            [
+                                '$binary' => [
+                                    'base64' => 'AAAAAAAAAAAAAAAAAAAAAA==',
+                                    'subType' => '04',
+                                ],
+                            ],
+                        ],
+                        'bsonType' => 'objectId',
+                        'algorithm' => 'AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic',
+                    ],
+                ],
+            ],
+            'bsonType' => 'object',
+        ],
+    ]],
+    ['bypassAutoEncryption' => true],
+    ['extraOptions' => ['mongocryptdBypassSpawn' => true]],
+];
+
+foreach ($tests as $driverOptions) {
+    $manager = new MongoDB\Driver\Manager(null, [], ['autoEncryption' => $driverOptions + $baseOptions]);
+    var_dump($manager);
```

Comment:
I'd propose we remove this and change the expected output block to:

```
--EXPECT--
===DONE===
```

Looking at the current `var_dump()` output, there's nothing relevant to the `autoEncryption` actually being applied, so I think all we're testing for here is that an exception hasn't been thrown. We do have other tests that do the same (i.e. no output expected to demonstrate the code runs successfully).